lib/agent.js

```diff
@@ -170,7 +167,7 @@ Agent.prototype.start = function (opts) {
       startTrace: stackObj.stack.split(/\n */).slice(1),
       main: pkg && pkg.main,
       dependencies: pkg && pkg.dependencies,
-      conf: this._conf
+      conf: this._conf.toJSON()
```

Comment:
This and the `toJSON()` implementation are to avoid a large log.trace dump to the console. I'm happy to pull out this change to a separate PR if you prefer.  This gist shows the the before/after of that log dump: https://gist.github.com/trentm/59864b57fb91388e061e5a64c9e8e80e